.travis.yml

```diff
@@ -17,6 +17,7 @@ before_install:
   - git checkout $TRAVIS_BRANCH
   - curl --output morula --location --fail https://github.com/Originate/morula/releases/download/0.1/morula-linux-amd64 && chmod +x morula
   - npm install -g yarn
+  - (cd exorelay-javascript && bin/setup && node_modules/.bin/build)
```

Comment:
Either `exo setup` or `exo run`/`exo test` should run build. There should be no need to run it manually.

Sounds like you need to add `exorelay` to the `before` block of `morula.yml`?